pokemon-showdown

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 'use strict';
 
+const spawnSync = require('child_process').spawnSync;
```

Comment:
Anyway, you still use `require('child_process').spawnSync` earlier, so at least be consistent. >:(